addon/components/sl-menu.js

```diff
@@ -396,6 +403,22 @@ export default Ember.Component.extend({
     }.on( 'init' ),
 
     /**
+     * Initialize initClassNames array
+     *
+     * @function initClassNames
+     * @observes "init" event
+     * @returns  {undefined}
+     */
+    initClassNames: function() {
+        var classNames = this.get( 'classNames' ),
```

Comment:
Per our discussion we are standardizing on not aligning var assignments equal signs.